src/Zend/Form.php

```diff
@@ -1157,7 +1157,7 @@ public function addElements(array $elements)
                 } else {
                     switch ($argc) {
                         case 0:
-                            continue;
+                            continue 2;
```

Comment:
This throws a warning in PHP 7.3 since `continue` in a `switch` statement is interpreted as a `break`.

In this particular case, the `switch` lives in a `foreach` loop, so it's possible that the original author intended this to be a `continue` instead of a `break`.  However, either construct would work here, as there's no executable code in the remainder of the `foreach` loop if this `switch` was broken at this point.  I changed it to a `continue 2;` to better show the intent and reduce confusion (also to remove the warnings).